src/include/rbd/librbd.hpp

```diff
@@ -173,6 +180,13 @@ class CEPH_RBD_API RBD
   int group_image_list(IoCtx& io_ctx, const char *group_name,
 		       std::vector<group_image_status_t>& images);
 
+  int group_snap_create(IoCtx& io_ctx, const char *group_name,
+			const char *snap_name);
+  int group_snap_remove(IoCtx& io_ctx, const char *group_name,
+			const char *snap_name);
+  int group_snap_list(IoCtx& group_ioctx, const char *group_name,
```

Comment:
Minor: should have support for rename as well.

Reply:
Rename will be addressed in a separate PR.

Reply:
https://github.com/ceph/ceph/pull/12431

Reply:
Yup -- that was a very old comment.